### PR TITLE
Update add.js to not allow negative minutes/hours and minutes greater than 60

### PR DESCRIPTION
**Pull request by TiyaZhan**

### CODE REVIEW

**Review by mogeritz — APPROVED** [review]

**Review by melvyn9 — APPROVED** [review]
Really good job on this @TiyaZhan! Looks like lab9 was really useful to handle these error handling cases. Can you please wait until pull request #85 is merged into main before you merge it please.

File: source/scripts/add.js
Comment (vchou105) [inline source/scripts/add.js:221]: I would capitalize class name per coding convention but we never explicitly wrote it in our coding protocol so I will leave the decision up to you.

**Review by vchou105 — COMMENTED** [review]
Can someone explain why there is MIME type error only on this branch? I'm not sure if the github pages branch interefered or anything but all the javascript and css files have a content type of text/html as opposed to their respective content types, so the changes made are unable to be run/tested properly.

**Review by MichaelRuddy — APPROVED** [review]
Great Job communicating with the team and glad you knocked this out

**Pull request merged**
